views/components/Game.jsx

```diff
@@ -1,18 +1,47 @@
-import React from 'react'
-import GameScene from './GameScene'
-import Map from './Map'
-import Character from './Character.jsx'
+import React from 'react';
+import GameScene from './GameScene';
+import Map from './Map';
+import Character from './Character';
+import * as BrowserActions from '../actions/Browser';
+
+class Game extends React.Component {
+  componentWillMount() {
+    this.resizeGameScene();
+  }
+
+  componentDidMount() {
+    this.props.dispatch(
+      BrowserActions.addEventListener('Game', 'resize', this.resizeGameScene)
+    );
+  }
+
+  resizeGameScene = () => {
+    this.props.dispatch(
+      BrowserActions.updateViewportSize()
```

Comment:
That's right, I wanted to discuss it with you. It maybe convenient to pass this property separately from the configuration so we can consume it from the browser store.